pkg/manifest/anaconda_installer_iso_tree.go

```diff
@@ -145,6 +154,11 @@ func (p *AnacondaInstallerISOTree) getBuildPackages(_ Distro) []string {
 	default:
 	}
 
+	if p.ISOBoot == Grub2ISOBoot {
+		// Needed for the i386-pc directory of modules needed by grub2 BIOS booting
```

Comment:
Perhaps it is wise to assert architecture here as well? Not super necessary of course.

Reply:
I assume the caller only sets it on x86. Otherwise we end up needing to figure out an error path, or... add a panic which I really really do not like.